src/modules/TeamsList/components/Teams/index.tsx

```diff
@@ -1,26 +1,37 @@
 import React, { FC } from 'react';
-import { PageTitle } from 'typography';
 
 import { TeamsHeader } from './components';
 import { MyTeam } from './components';
 import { Team, TeamList } from 'types';
 import { TeamItem } from './components';
+import { TableTitle } from 'modules/StudentsTable/styled';
+import { Button } from 'typography';
+import { TeamsTitleWrapper } from 'modules/TeamsList/styled';
 
 type TeamsProps = {
   teams: TeamList;
   title?: string;
   myTeam?: Team;
   userId: string;
+  isAdmin: boolean;
+  onClickSortStudents: () => void;
 };
 
 export const Teams: FC<TeamsProps> = ({
   title = 'Teams',
   teams,
   myTeam,
   userId,
+  isAdmin,
+  onClickSortStudents,
 }) => (
   <>
-    <PageTitle>{title}</PageTitle>
+    <TeamsTitleWrapper>
+      <TableTitle>{title}</TableTitle>
+      {isAdmin ? (
```

Comment:
Why only admin can sort students? or it's not a general sorting?

Reply:
Only admin can sort students

Reply:
But {isAdmin && <Button onClick={onClickSortStudents}>SORT STUDENTS</Button>} can be used there